workshops/parallax_effect/README.md

```diff
@@ -0,0 +1,183 @@
+---
+name: 'Parallax Effect'
+description: 'Creating greater user experience on web using javaScript'
+author: '@giridhar7632'
+---
+
+# Parallax Effect
+**creating illusion of depth using javaScript**
```

Comment:
this is a weird format

Reply:
I thought It would be nice if there is a sub-title
I have deleted it.